requirements.txt

```diff
@@ -1 +1,2 @@
-django==2.0
\ No newline at end of file
+django==2.0
+beautifulsoup4==4.6.0
```

Comment:
@pvacca97 Why do you need this library?

Reply:
Like I don't understand why you need to pull data from HTML and XML in this PR

Reply:
I thought I was going to use this library but that didn't end up being the case... Might still be useful later on when we want to manipulate the homepage welcoming the user signing in?

Reply:
It might be best to remove it until then and add it later if the need arises.